src/test/resources/network/brightspots/rcv/test_data/generic_csv_test/generic_csv_test_cvr.csv

```diff
@@ -1,4 +1,4 @@
-Response ID,Lettuce,Broccoli,Cucumber,Cauliflower,Write-In
+,Lettuce,Broccoli,Cucumber,Cauliflower,Write-In
```

Comment:
Can we please keep this as-is, and make a new file and test for this case instead?